src/test/resources/features/meeting.feature

```diff
@@ -28,16 +28,16 @@ Feature: Meetings
 
   Scenario: Remove a meeting
     Given I navigate to Available section
-      And I create a Meeting with the following information: "Ariel Rojas", "meeting", "15:00", "16:00", "test@exchange.com", "Important"
+      And I have a Meeting with the following information: "Ariel Rojas", "meeting", "15:00", "16:00", "test@exchange.com", "Important"
     When I remove the Meeting
     Then an information message should be displayed "Meeting successfully removed"
       And the meeting should not be displayed in the Schedule bar
       And the meeting information should not be displayed in the Next section of Main page
-      And the Meeting should be listed in the Meetings of Room using the API
+      And the Meeting should not be listed in the meetings of Room using the API
 
   Scenario: Update a meeting
     Given I navigate to Available section
-      And I create a Meeting with the following information: "JeanCarlo.Rodriguez", "meeting", "16:00", "17:00", "test@exchange.com", "this meeting is update"
+      And I have a Meeting with the following information: "JeanCarlo.Rodriguez", "meeting", "16:00", "17:00", "test@exchange.com", "this meeting is update"
```

Comment:
you create this for API right? change the name of this step to reflect that action